app/uk/gov/hmrc/plasticpackagingtax/registration/models/genericregistration/PartnershipDetails.scala

```diff
@@ -82,18 +80,7 @@ case class PartnerPartnershipDetails(
   partnershipType: PartnerTypeEnum,
   partnershipName: Option[String] = None,
   partnershipBusinessDetails: Option[PartnershipBusinessDetails] = None
-) {
-
-  lazy val name: Option[String] = partnershipType match {
```

Comment:
This one is slight interesting but computer seems to say yes.
I have replace the only UI usagge with a call to partner.name.